### DISCUSSION TITLE
Does model instancing allow for large polycounts on mobile?

### DISCUSSION THREAD

**Discussion by andrewmanq** [google/model-viewer#4258] [Q&A]
I've got some plant models that exceed 100k vertices, but the branches are instanced -- it registers under 4000 triangles in the editor. Do mobile browsers have the hardware acceleration capable of instancing stuff? If so, these models will look substantially better.

**Comment by elalish** [comment]
It's part of WebGL2 and an extension in WebGL1, so I think it has decently broad support by now. Give it a try and let us know what you find!
